### ISSUE TITLE
lslint binaries not executable on Linux and Mac

### ISSUE THREAD

**Issue by thickbrick** [buildersbrewery/sublime-lsl#1]
### Expected behavior

lslint should run.

### Actual behavior

lslint doesn't run (errno 13)
Makopo/siblime-text-lsl does the chmod in init.py, but not sure that's needed (vs. just distributing the files with the right file permissions.)

### Environment

```text
=BB= LSL:

    version: 1.2.0
    installed via Package Control: True

Sublime Text:

    channel: stable
    build: 3143
    portable: True
    platform: Linux
    architecture: x64
```

**Comment by buildersbrewery** [comment]
Thanks for the input. Will try to update by tonight.

**Comment by buildersbrewery** [comment]
reported at Makopo/lslint#66

**Comment by buildersbrewery** [comment]
fixed in `v1.2.1`, please give Package Control an hour or so to update.

**Comment by buildersbrewery** [comment]
@thickbrick I hope this is fixed now, otherwise please re-open.

---

I've got a couple thousand lines still pending for tooltip data. Also, the syntax still has issues. Working on it.

In case you find something, I'd appreciate a report.
